midend/lib/Conversion/ConvOptimization/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
 add_mlir_library(ConvOptimization
 	ConvOptimize.cpp
+  ConvNhwcFhwcOptimize.cpp
+  ConvNhwcFhwcOptimizeTile.cpp
```

Comment:
Now that we have distinguished between the two different methods `ConvNhwcFhwcOptimize.cpp` and `ConvNhwcFhwcOptimizeTile.cpp`, and also distinguished between `ConvNhwcFhwcOptimize` and the optimization of ConvNchwFchw (i.e. `ConvOptimize.cpp`), would it be better to unify the naming convention to `{NAME}{METHOD}Optimize.cpp`?

This means that you need to change the naming conventions of all `ConvOptimize`, `ConvNhwcFhwcOptimize` and `ConvNhwcFhwcOptimizeTile` passes.